changelog.d/5-internal/play-with-version-types

```diff
@@ -0,0 +1 @@
+Introduce VersionNumber newtype (see `/libs/wire-api/test/unit/Test/Wire/API/Routes/Version.hs` for explanation)
```

Comment:
I expected the mentioned file to include some comment explaining the need and usage of this wrapper.

Reply:
fair!  :)

a8d8ed7f1b48230e9efb7fb94fb29ee1d09e87d0 ?